FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/DataProcessor/ParquetDataProcessor.cs

```diff
@@ -17,7 +17,8 @@
 using Microsoft.Health.Fhir.Synapse.Common.Models.Data;
 using Microsoft.Health.Fhir.Synapse.Core.DataProcessor.DataConverter;
 using Microsoft.Health.Fhir.Synapse.Core.Exceptions;
-using Microsoft.Health.Fhir.Synapse.Parquet.CLR;
+using Microsoft.Health.Fhir.Synapse.Parquet;
```

Comment:
Is neccessary to locate `Parquet` under the namespace `Synapse`?

Reply:
Fixed. Thanks